### DISCUSSION TITLE
Create Fritzing Parts for Seeed XIAO MCUs

### DISCUSSION THREAD

**Discussion by meilily** [Seeed-Studio/OSHW-XIAO-Series#7] [Accomplished]
[Fritzing](http://fritzing.org/) is one of the most popular open-source software tools that simplifies the process of designing electronic circuits and creating printed circuit boards (PCBs). It provides a user-friendly interface for drawing circuit diagrams, placing components on a virtual breadboard, and designing custom PCB layouts. With its interactive features and extensive component library, Fritzing is suitable for beginners and experienced electronics enthusiasts alike. Currently, we have some Fritzing parts for XIAO, created by the amazing member Peter Aka [Vanepp](https://forum.fritzing.org/u/vanepp) at the Fritzing forum. Here are the available parts created by Peter:

- [Seeed Studio XIAO SAMD21](https://forum.fritzing.org/t/seeeduino-xiao-anyone-got-the-part/10820/11)
- [Seeed Studio XIAO ESP32C3](https://forum.fritzing.org/t/seeeduino-xiao-anyone-got-the-part/10820/11)
- [Seeed Studio XIAO nRF52840 (Sense)](https://forum.fritzing.org/t/seeed-studio-xiao-nrf52840/18403/3)

However, we're still missing some parts of the Seeed Studio XIAO Series as follows. *And we need the community power to help create the rest Fritzing parts. Please comment to share the XIAO Fritzing parts that you created.*  Thanks!💚

- [Seeed Studio XIAO ESP32C6](https://wiki.seeedstudio.com/xiao_esp32c6_getting_started/)
- [Seeed Studio XIAO ESP32S3 (Sense)](https://wiki.seeedstudio.com/xiao_esp32s3_getting_started/)
- [Seeed Studio XIAO RP2040](https://wiki.seeedstudio.com/XIAO-RP2040/)

P.S. We have a repo to host all Fritzing Parts of Seeed products, and all the XIAO Fritzing parts will be merged to [this repo here](https://github.com/Seeed-Studio/fritzing_parts). Here you can find more guides on how to [create Fritzing parts](https://fritzing.org/learning/tutorials/creating-custom-parts)

**Comment by gokuxmaker** [comment]
![image](https://github.com/Seeed-Studio/XIAO_Series/assets/64944561/80ff5f81-0706-4fa4-8caa-51cb8d7e6b69)

**Reply by meilily** [reply]
💚 A big shout-out to @gokuxmaker for creating the fritzing parts for XIAO SAMD21, XIAO RP2040, XIAO nRF52840, XIAO nRF52840 Sense, XIAO ESP32C3, XIAO ESP32S3, XIAO ESP32S3 Sense, and also XIAO ESP32C6. All are available on this repo file: https://github.com/Seeed-Studio/fritzing_parts/tree/master/XIAO%20Boards

P.S. @gokuxmaker also created this super cool poster, and also a video tutorial on how to import these parts to Fritzing. You can learn more here: https://www.seeedstudio.com/blog/2024/07/11/xiao-fritzing-parts/

![XIAO Fritzing Parts](https://github.com/Seeed-Studio/OSHW-XIAO-Series/assets/111474938/996ea1f7-f7c3-422e-81db-e11a28da44d9)
